uni/pubspec.yaml

```diff
@@ -7,74 +7,80 @@ publish_to: "none" # We do not publish to pub.dev
 # To change it manually, override the value in app_version.txt.
 # The app version code is automatically also bumped by CI.
 # Do not change it manually.
-version: 1.7.22+235
+version: 1.8.0-beta.19+252
```

Comment:
Is it supposed to have the beta version here? (this is what will go to the master branch)

Reply:
Yes, the CI will fix it